app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2360,6 +2364,29 @@ class BrowserTabFragment :
         }
     }
 
+    private fun launchSharePromoRMFPageChooser(url: String, shareTitle: String) {
+        val share = Intent().apply {
+            action = Intent.ACTION_SEND
+            putExtra(Intent.EXTRA_TEXT, url)
+            putExtra(Intent.EXTRA_TITLE, shareTitle)
+            type = "text/plain"
+        }
+
+        val pi = PendingIntent.getBroadcast(
+            requireContext(),
+            0,
+            Intent(requireContext(), SharePromoLinkRMFBroadCastReceiver::class.java).apply {
+                putExtra(EXTRA_MESSAGE_ID, viewModel.remoteMessageId())
+            },
+            PendingIntent.FLAG_UPDATE_CURRENT or PendingIntent.FLAG_IMMUTABLE,
+        )
+        try {
+            startActivity(Intent.createChooser(share, null, pi.intentSender))
```

Comment:
I see title here is `null`. I remember you mentioned that not everyone support title, maybe we can try with this one.

Reply:
I tried it and it didn't work. 
Also, if you go to the documentation of this method, you can see in the description of the parameter `title`: _"Optional title that will be displayed in the chooser, only when the target action is **not ACTION_SEND** or ACTION_SEND_MULTIPLE."_